Source/Core/Core/Config/MainSettings.cpp

```diff
@@ -222,6 +222,8 @@ const Info<bool> MAIN_PRECISION_FRAME_TIMING{{System::Main, "Core", "PrecisionFr
                                              DEFAULT_PRECISION_FRAME_TIMING};
 const Info<float> MAIN_OVERCLOCK{{System::Main, "Core", "Overclock"}, 1.0f};
 const Info<bool> MAIN_OVERCLOCK_ENABLE{{System::Main, "Core", "OverclockEnable"}, false};
+const Info<float> MAIN_VI_FREQUENCY{{System::Main, "Core", "VIFrequency"}, 1.0f};
```

Comment:
It looks like this setting itself isn't really a `Frequency`. I think using "Overclock" phrasing like with the CPU "Overclock" would be better.

Reply:
The motivation for the name is here: https://github.com/dolphin-emu/dolphin/pull/13046#issuecomment-2838273475.

(Also note that the UI name for adjusting the CPU clock is "Emulated CPU Clock". There is no "Overclock" in the name.)

Reply:
I think it's fine for the UI-name to not perfectly match the INI-name. I think I'd still prefer `VIOverclock` in the ini file since the number is not a frequency. `Scale` or `Factor` or `Multiplier` would be okay, but `VIOverclock` is consistent with the Core `Overclock` setting name.